templates/zerver/emails/confirm_new_email.source.html

```diff
@@ -5,25 +5,9 @@
 {% endblock %}
 
 {% block content %}
-<p>
-    {% trans %}
-    Hi {{ user_name }},
-    {% endtrans %}
-</p>
+<p>{% trans %}Hi {{ user_name }},{% endtrans %}</p>
 
-<p>
-    {% trans %}
-    We received a request to change the email address for the Zulip account on
-    {{ realm_uri }} from {{ old_email }} to {{ new_email }}.
-    To confirm this change, please click below:
-    {% endtrans %}
-    <a class="button" href="{{ activate_url }}">{{_('Confirm email change') }}</a>
-</p>
+<p>{% trans %}We received a request to change the email address for the Zulip account on{{ realm_uri }} from {{ old_email }} to {{ new_email }}.To confirm this change, please click below:{% endtrans %}<a class="button" href="{{ activate_url }}">{{_('Confirm email change') }}</a></p>
```

Comment:
there are a few spaces missing here. Also each translation piece can be on its own line, e.g. 
`<a class="button" href="{{ activate_url }}">{{_('Confirm email change') }}</a>` 
doesn't have to be on the same line as the rest of it.